controllers/numaresourcesoperator_controller.go

```diff
@@ -334,8 +339,40 @@ func (r *NUMAResourcesOperatorReconciler) deleteUnusedDaemonSets(ctx context.Con
 				if err := r.Client.Delete(ctx, &ds); err != nil {
 					klog.ErrorS(err, "error while deleting daemonset", "DaemonSet", ds.Name)
 					errors = append(errors, err)
+				} else {
+					klog.V(3).Infof("DS[%s] deleted", ds.Name)
```

Comment:
let's use the full "Daemonset" (or "DaemonSet") here

Reply:
sure.